src/ucp/core/ucp_ep.h

```diff
@@ -195,6 +195,7 @@ struct ucp_ep_config_key {
     ucp_lane_index_t         tag_lane;        /* Lane for tag matching offload (can be NULL) */
     ucp_lane_index_t         wireup_msg_lane; /* Lane for wireup messages (can be NULL) */
     ucp_lane_index_t         cm_lane;         /* Lane for holding a CM connection (can be NULL) */
+    ucp_lane_index_t         keepalive_lane;  /* Lane for checking a connection */
```

Comment:
maybe `Lane for checking a connection state`? Also worth mentioning that it can be NULL (like for other lanes)

Reply:
done